src/controllers/statsPages/darknodeStatsPage/DarknodeStatsPage.tsx

```diff
@@ -171,6 +171,7 @@ export const DarknodeStatsPage = () => {
         <Stats className="overview--bottom--right">
           {/* <Stat message="All time total" big>$?</Stat> */}
           <Stat
+            className="darknode-cli"
             message="Reward Period/EPOCH Ends"
```

Comment:
Should EPOCH be capitalized here?

Reply:
Right, Vinnie decided to put "Epoch" everywhere.